src/data/injections.js

```diff
@@ -213,7 +213,7 @@ const AVAILABLE_INJECTIONS = [
   },
   {
     id: "bug1567610",
-    platform: "desktop",
+    platform: "all",
```

Comment:
Was this also an intended change for this patch?

Reply:
yeah, I was able to test this properly after building Firefox on Android, so the fix works fine